app/assets/v2/js/cart.js

```diff
@@ -1,5 +1,26 @@
-let grantHeaders = [ 'Grant', 'Amount', 'Type', 'Total CLR Match Amount' ];
-let grantData = [];
+/**
+ * @notice Vue component for managing cart and checkout process
+ * @dev If you need to interact with the Rinkeby Dai contract (e.g. to reset allowances for
+ * testing), use this one click dapp: https://oneclickdapp.com/drink-leopard/
+ */
+
+// Constants
+const BN = web3.utils.BN;
+const ETH_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE';
+const MAX_UINT256 = '115792089237316195423570985008687907853269984665640564039457584007913129639935';
+const DEV_GRANT_ADDRESS = '0x0000000000000000000000000001000000000000'; // rinkeby grants seem unable to accept ETH, so use dummy address
```

Comment:
The Rinkeby grant contracts aren't verified on Etherscan and I haven't seen the grant contracts yet. But the tx reverts with the regular grant address, so I'm guessing the Rinkeby ones don't accept ETH via the fallback function. I believe it should though, because in [fund.js](https://github.com/gitcoinco/web/blob/master/app/assets/v2/js/grants/fund.js#L269) it looks like they use an ordinary send for ETH donations.

I figured this doesn't matter too much so just chose an unused address to bypass this for now

Reply:
Got it, weird. Let's check with the GC folks about this, just to make sure something isn't off that will effect prod.

Assuming you're correct, let's change the address here to `0x058e6afe3590A7DA1A5Ad7A6fE0df7060c7d12cF` which is an address we can use on Rinkeby to recycle the funds. Not that it's a huge deal to lose tesnet ETH but 🤷‍♂️

Reply:
Sure, will do. But what address is that? I don't see it in the codebase anywhere